Resources/config/services.yml

```diff
@@ -14,6 +15,11 @@ services:
         arguments:
             - @support_tools.info_collector.system_info.ezc_system_info
 
+    support_tools.info_collectors.json_composer_lock:
```

Comment:
Naming (same that for php and ezc): `support_tools.info_collectors.composer.lock_file`.